cd2h_repo_project/modules/theme/templates/cd2hrepo_theme/trackingcode.html

```diff
@@ -1 +1,17 @@
-{# TODO: Insert our tracking code here #}
+{%- if config.ANALYTICS_URL %}
+<!-- Matomo -->
+<script type="text/javascript">
+  var _paq = window._paq || [];
+  /* tracker methods like "setCustomDimension" should be called before "trackPageView" */
+  _paq.push(['trackPageView']);
+  _paq.push(['enableLinkTracking']);
+  (function() {
+    var u="https://{{config.ANALYTICS_URL}}/";
+    _paq.push(['setTrackerUrl', u+'matomo.php']);
+    _paq.push(['setSiteId', '{{config.ANALYTICS_SITEID}}']);
+    var d=document, g=d.createElement('script'), s=d.getElementsByTagName('script')[0];
+    g.type='text/javascript'; g.async=true; g.defer=true; g.src='//cdn.matomo.cloud/{{config.ANALYTICS_URL}}/matomo.js'; s.parentNode.insertBefore(g,s);
+  })();
+</script>
+<!-- End Matomo Code -->
+{%- endif -%}
```

Comment:
Can you add a link to the matomo documentation for this? Either in the code or as a comment in the issue is fine. I found the set up for most of this, but can not find any reference to the cdn, "//cdn.matomo.cloud/{{config.ANALYTICS_URL}}/matomo.js", can you point me to it?

Reply:
Here are the [docs](https://matomo.org/docs/) and [FAQs](https://matomo.org/faq/).

Reply:
Added info to indicate where the tracking code is found.

Reply:
Thank you.